src/api/routes/tx.ts

```diff
@@ -286,6 +287,27 @@ export function createTxRouter(db: DataStore): express.Router {
     })
   );
 
+  router.get(
+    '/count',
+    cacheHandler,
+    asyncHandler(async (req, res) => {
+      const from = parseTimestampQueryInput(req.query.from);
+      const to = parseTimestampQueryInput(req.query.to);
```

Comment:
Would it make better sense to ask for YYYY-MM-DD dates rather than timestamps? With these args, you'd get incomplete day counts on the boundaries that you requested.

Reply:
Starting to sound like a bit of a, well a bit of a dick, however im kinda a lead buy example type of guy, so im game now how do you recommend going about that